doc/source/whatsnew/v0.24.0.txt

```diff
@@ -189,6 +189,7 @@ Other Enhancements
 
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- :func:`DataFrame.to_csv` now uses :func:`os.linesep` rather than ``'\n'`` for the default line terminator.(:issue:`20353`)
```

Comment:
can you elabortate on what if anything a user needs to do (e.g. when should they care about this)